tests/shared.py

```diff
@@ -0,0 +1,45 @@
+class MockArgs:
+    def __init__(self, model):
+        self.model = model #part of model config
+        self.model_norm = "gain_only_layer_norm"
+        self.rotary_old = False
+        self.qk_norm = False
+        self.train_data = ["/gscratch/cse/jrfish/open_lm/tests/shard_00000000.tar",]
```

Comment:
instead of pointing to local path can we store this is huggingface or something and then pull it automatically if someone runs unit tests?

Reply:
here is some sample data that could be used for testing: https://huggingface.co/datasets/mlfoundations/open_lm_example_data/blob/main/validation_data/shard_00000000.tar